apps/ledger-live-mobile/android/app/src/main/jni/MainApplicationTurboModuleManagerDelegate.cpp

```diff
@@ -0,0 +1,38 @@
+#include "MainApplicationTurboModuleManagerDelegate.h"
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)